src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py

```diff
@@ -386,10 +388,10 @@
     text: az postgres flexible-server migration update --subscription xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx --resource-group testGroup --name testserver --migration-name testmigration --db-names db1 db2
   - name: Allow the migration workflow to overwrite the DB on the target.
     text: az postgres flexible-server migration update --subscription xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx --resource-group testGroup --name testserver --migration-name testmigration --overwrite-dbs
-  - name: Cut-over the data migration for either all databases or few selected ones. After this is complete, subsequent updates to the selected databases will not be migrated to the target.
-    text: az postgres flexible-server migration update --subscription xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx --resource-group testGroup --name testserver --migration-name testmigration --cutover db1 db2 db3
```

Comment:
can we not cuto-over specific databases now?

Reply:
No, we dont want to expose that functionality as of now via CLI.

Reply:
was this ever released? how many customers will be impacted due to this change?

Reply:
The functionality as a whole was broken. Since cutover is for online only and we never did private preview for online, no customers should be impacted.